enterprise_access/apps/subsidy_request/tasks.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tasks for subsidy requests app.
+"""
+
+import logging
+
+from celery import shared_task
+from django.apps import apps
```

Comment:
use of `apps.get_model` here prevents us from having a circular import (models.py <--> tasks.py)